services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/logging/CheckLoggingSpec.groovy

```diff
@@ -1,9 +1,69 @@
 package org.openkilda.functionaltests.spec.logging
 
+import groovy.util.logging.Slf4j
 import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.testing.service.elastic.ElasticQueryBuilder
+import org.openkilda.testing.service.elastic.ElasticService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.client.HttpClientErrorException
 import spock.lang.Narrative
 
+@Slf4j
 @Narrative("This specification ensures that all logging facilities are up and running after Kilda deployment")
 class CheckLoggingSpec extends BaseSpecification {
 
+    @Autowired
+    ElasticService elasticService
+
+
+    def "Check Floodlight logging"() {
+        when: "Retrieve floodlight logs for last 5 minutes"
+        def tagName = "kilda-floodlight"
+        def result = elasticService.getLogs(new ElasticQueryBuilder().setTags(tagName).setTimeRange(300).build())
+        assert result?.hits?.total > 0
+
+        then: "There should be discovery messages"
+        result.hits.hits.any {hit -> hit.source.message.toLowerCase().contains("push discovery package via")}
```

Comment:
Maybe it is worth moving "push discovery package via" to some constants.

Reply:
May well be.